### ISSUE TITLE
Incorrect role is defined as link for ‘tab item’ Understand ‘OData in 6 steps’ page.

### ISSUE THREAD

**Issue by habbes** [OData/odataorg.github.io#266]
If Incorrect role is defined than screen reader user not able to access the control as expect incorrect behavior according to defined and user will get confused. 

## Repro-Steps:

- Hit the URL https://www.odata.org/ to open OData website
- Tab till 'Developers' link and press enter
- Tab till 'Getting Started' option and press enter
- Tab till 'Understand OData in 6 steps' link and press enter
- Observe whether role property is properly defined for tab control or not.

### Actual Result:

Incorrect role is defined as link for ‘tab item’ Understand ‘OData in 6 steps’ page. Role is defined as link.

### Expect Result:

Correct role should be defined as ‘tab item’ Understand ‘OData in 6 steps’ page, so that user can get the proper information related to control.

### Note:

1.Similar issue exists throughout the application for same control.
